pkg/skaffold/deploy/helm/deploy.go

```diff
@@ -253,7 +253,13 @@ func (h *Deployer) Render(ctx context.Context, out io.Writer, builds []build.Art
 	renderedManifests := new(bytes.Buffer)
 
 	for _, r := range h.Releases {
-		args := []string{"template", r.ChartPath}
+		var chartPath string
+		if r.LocalChartPath != "" {
+			chartPath = r.LocalChartPath
+		} else {
+			chartPath = r.RemoteChartPath
+		}
```

Comment:
How about reducing this or adding a utility function which 
```suggestion
		chartPath := r.RemoteChartPath
		if r.LocalChartPath != "" {
			chartPath = r.LocalChartPath
		}
```

e.g. 
```
func getChartPath(h latest.HelmRelease) string {
   if h.LocalChartPath != "" {
      return h.LocalChartPath
   } 
  return r.RemoteChartPath
}

```

Reply:
done